lib/linguist/languages.yml

```diff
@@ -2337,6 +2338,7 @@ JSON:
   - ".tern-project"
   - ".watchmanconfig"
   - composer.lock
+  - form.4DForm
```

Comment:
This shouldn't be necessary as you already added `.4DForm` as a JSON extension.

Reply:
Ok I was not sure If I had to add it in the filename too.